apps/nextjs-website/src/components/organisms/Auth/ConfirmSignUp.tsx

```diff
@@ -32,16 +30,22 @@ const ConfirmSignUp = ({ email, onBack }: ConfirmSignUpProps) => {
               <IconInbox />
             </Stack>
             <Typography variant='h4' pt={5} mb={4} textAlign='center'>
-              {confirmSignUp.confirmSignUp}
+              {confirmSignUp('title')}
             </Typography>
             <Typography
               variant='body2'
               mb={2}
               dangerouslySetInnerHTML={{
-                __html: confirmSignUp.description(email),
+                __html: confirmSignUp
+                  .raw('emailSent')
+                  .replace('{email}', email),
```

Comment:
I think is here is better to use [parametric translation](https://next-intl-docs.vercel.app/docs/usage/messages) of `next-intl` instead of replace